### PR TITLE
Solidity testing and refactor

### PR DESCRIPTION
**Pull request by hkalodner**

### CODE REVIEW

**Review by joshuacolvin0 — APPROVED** [review]
LGTM

File: packages/arb-bridge-eth/contracts/bridge/Bridge.sol [hunk 1]
```diff
@@ -84,7 +103,7 @@ contract Bridge is OwnableUpgradeable, IBridge {
         address destAddr,
         uint256 amount,
         bytes calldata data
-    ) external override returns (bool success, bytes memory returnData) {
+    ) external virtual override returns (bool success, bytes memory returnData) {
```
Comment (DZGoldman) [inline packages/arb-bridge-eth/contracts/bridge/Bridge.sol:106]: Is this necessary? looks like it isn't overridden by BridgeMock (or is it "preemtive" for future testing?)

**Review by DZGoldman — APPROVED** [review]

File: packages/arb-bridge-eth/contracts/bridge/Bridge.sol (hunk 1 above)
Comment (hkalodner) [reply packages/arb-bridge-eth/contracts/bridge/Bridge.sol:106]: Ah, I think this was an accident. Great catch

**Review by DZGoldman — APPROVED** [review]
lgtm

**Pull request merged**